git_machete/cmd.py

```diff
@@ -283,7 +283,7 @@ def up(b, prompt_if_inferred):
         u = infer_upstream(b)
         if u:
             if prompt_if_inferred:
-                if ask_if("Branch '%s' not found in the tree of branch dependencies; rebase onto the inferred upstream '%s'? [y/n] " % (b, u)):
```

Comment:
I would like to ask you to start limiting length to 80 chars. It is tricky to review on GitHub without vertical scrollbar which is on the bottom.

Reply:
Done, I limited all line lengths in `traverse` method.

Reply:
Nice, thx!